src/Tribe/Editor/REST/V1/Endpoints/Single_Ticket.php

```diff
@@ -263,6 +263,7 @@ public function add_ticket( WP_REST_Request $request, $nonce_action ) {
 			'ticket_end_date' => $body['end_date'],
 			'ticket_end_time' => $body['end_time'],
 			'ticket_sku' => $body['sku'],
+			'ticket_iac' => $body['iac'],
```

Comment:
:warning: Array double arrow not aligned correctly; expected 14 space(s) between "'ticket_iac'" and double arrow, but found 1.

<sub>hash: aa596115452417c577fb4d42482a08d9</sub>